.travis.yml

```diff
@@ -20,7 +20,9 @@ cache:
 python:
 - 3.7
 
-os: linux
+os:
+  - linux
+  - windows
```

Comment:
Can we run `just` the unit tests, and not the whole build?

Reply:
Yep. `language: python` is not supported under windows. I will need to separate the jobs and run the full build under linux and unit test under windows with `language: bash`.